eng/common/scripts/stress-testing/java11-mvn-mariner-dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM mcr.microsoft.com/openjdk/jdk:11-mariner as builder
```

Comment:
We can't check in files to `eng/common` as they are synced from azure-sdk-tools to all repos.